README.md

```diff
@@ -48,7 +48,9 @@ Before extending into new frontiers, we need to improve the loaders API enough t
 
 ### Milestone 2: Usability improvements
 
-- [ ] Provide a way to register loaders without a command-line flag, for example via a `"loaders"` field in `package.json` ([#98](https://github.com/nodejs/loaders/issues/98) or https://github.com/nodejs/node/pull/43973).
+- [ ] Provide a way to register loaders without a command-line flag, for example via a `"nodeOptions"` field in `package.json` ([#98](https://github.com/nodejs/loaders/issues/98) or https://github.com/nodejs/node/pull/43973 or https://github.com/nodejs/node/pull/46534#issuecomment-1420041456).
+
+- [ ] Provide a way to register loaders programmatically, for example `import { registerLoader } from 'module'`.
```

Comment:
Shouldn't this be under `Parity with CommonJS` since with CommonJS you can monkey-patch to add a "loader" at runtime, for example `@babel/register`.

Reply:
The “real” distinction between Milestones 1 and 2 is that everything in 1 needs to land while the Loaders API is experimental (because each addition would be a breaking change) whereas everything in 2 can land after the API is stable, without causing a breaking change. It just so happened that most of the things that fall in the first category had to do with CommonJS parity.